### DISCUSSION TITLE
Is there a way to give editor options to the generateImage function?

### DISCUSSION THREAD

**Discussion by horiu** [epam/ketcher#1873] [Q&A]
Hi,

Thanks for developing Ketcher!

Is there a way to set the options `atomColoring: false` and `showHydrogenLabels: "Hetero"` when generating with the generateImage function?

Output image of generateImage function
![image](https://user-images.githubusercontent.com/75790328/204464352-0c757488-de2c-46c9-98b8-6e4673cd13ac.png)
Image I want
![image](https://user-images.githubusercontent.com/75790328/204464440-a856f32b-fe1d-4a10-bf65-0fae03d2b75d.png)

Thank you,

**Comment by thomastnj** [comment]
Hi,
I'm working on a project and need to be able to do the same thing. Can someone help answer the original question? Is it possible to set the rendering options that would be applied to the output image of the "generateImage" function? Calling ketcher.editor.setOptions only works for the structure image in the editor canvas.
Thank you!
